### PR TITLE
[bugfix] corrected unrolled loops ranges

### PR DESCRIPTION
**Pull request by lasofivec**
The ranges of the loops for the unrolling examples were off.
I am pretty sure it was only working because:

- the size of `X` and `Y` are divisible by 4
- and, the flag ``boundscheck=True``

### CODE REVIEW

File: tutorial.ipynb [hunk 1]
```diff
@@ -967,13 +967,13 @@
...
     "\n",
-    "    for i in range(0, k, 4):\n",
+    "    for i in range(0, k-rem, 4):\n",
     "        res += ((i + 1) * fabs(x[i] - y[i])\n",
     "               +(i + 2) * fabs(x[i + 1] - y[i + 1])\n",
     "               +(i + 3) * fabs(x[i + 2] - y[i + 2])\n",
     "               +(i + 4) * fabs(x[i + 3] - y[i + 3]))\n",
     "    \n",
-    "    for i in range(i, rem):\n",
+    "    for i in range(i, k):\n",
```
Comment (jeremiedbb) [inline tutorial.ipynb]: Actually it's in `range(k-rem, k)` because after the first loop `i = k - rem - 4`

**Review by jeremiedbb — COMMENTED** [review]
Thanks. I messed up with a previous implementation where I increased the pointers instead.

File: tutorial.ipynb (hunk 1 above)
Comment (lasofivec) [reply tutorial.ipynb]: True !

**Comment by jeremiedbb** [comment]
Thank @lasofivec !

**Pull request merged**